spec/models/user_spec.rb

```diff
@@ -462,12 +462,33 @@
   describe ".find_by_lower_email" do
     it "uses cache" do
       u = FactoryBot.build(:user_with_email)
-      expect(User.find_by_lower_email(u.email.upcase, u)).to eq(u)
+      Vmdb::Deprecation.silenced do
+        expect(User.find_by_lower_email(u.email.upcase, u)).to eq(u)
```

Comment:
if we are not calling this method anymore, then I wonder if we need to test it

Reply:
Yes, they should be removed anyway. I was trying to make sure these alias are working. Maybe I could just change them all to lookup_by_* ?

Reply:
quick hit suggested it is in a number of repos:

|count|repo|
|---|---|
| 68| manageiq|
| 35| manageiq-automation_engine|
| 23| manageiq-ui-classic|
| 12| manageiq-content|
|  8| manageiq-providers-openstack|
|  5| manageiq-providers-vmware|
|  4| manageiq-providers-azure|
|  3| manageiq-schema|
|  3| manageiq-providers-ovirt|
|  3| manageiq-providers-amazon|
|  2| manageiq-smartstate|
|  2| manageiq-api|
|  1| manageiq-providers-scvmm|
|  1| manageiq-pods|
|  1| manageiq-gems-pending|
|  1| manageiq-consumption|

So I'm thinking deprecation may be our best route.

Also, looks like they are defined in `manageiq`, `manageiq-proviers-azure` and `manageiq-providers-amazon`

---
/via
```
ag -l find_by_ manageiq* | awk -F/ '{print $5}' | sort | uniq -c | sort -nr

```

Reply:
Thanks for the list! I am working on those repos to use lookup_by_*. I will add deprecation warnings in another PR after I have done for these repos (or there will be a lot of deprecation warning showing up, which might be annoying for other developers)